### PR TITLE
Fix tick labels

### PR DESCRIPTION
**Pull request by asinghvi17**
This tries to account for the fact that we rescale the projected point by a factor of 100_000.

```julia
julia> fig = Figure(); ga = GeoAxis(fig[1, 1], coastlines = true, dest = "+proj=ortho", lonlims = (-90, 90), latlims = (-90, 90)); fig

julia> MakieLayout.deactivate_interaction!(ga, :rectanglezoom)

julia> point = select_point(ga.scene)
Observable{Point{2, Float32}} with 0 listeners. Value:
Float32[0.0, 0.0]

julia> on(point) do val
           println(val)
       end
ObserverFunction defined at REPL[24]:2 operating on Observable{Point{2, Float32}} with 1 listeners. Value:
Float32[0.0, 0.0]

julia> Float32[-41.442448, -12.973355]
Float32[-9.051716, -11.688086]
Float32[35.24889, -5.3072042]
Float32[56.347336, -4.200298]
Float32[56.347336, 21.104654]
Float32[46.330616, 36.424725]
Float32[33.74524, 45.28347]
Float32[48.534336, 5.2599907]
Float32[20.249252, 5.2599907]
Float32[-11.216833, 5.2599907]
Float32[-45.319996, 4.805335]
Float32[-55.19109, 2.1018956]
Float32[-59.652912, -0.70646304]
Float32[-57.19303, -14.811205]
Float32[-57.19303, -22.218555]
Float32[-33.21361, -41.613014]
Float32[-10.634323, -49.085068]
Float32[-2.6791973, -60.19085]
Float32[13.56968, -60.19085]
Float32[28.611433, -57.53987]
Float32[35.327843, -47.53925]
Float32[43.282967, -39.25931]
Float32[52.101334, -29.335627]
Float32[52.534706, -17.462183]
Float32[60.054707, -2.740162]
Float32[60.054707, 8.52125]
Float32[60.054707, 14.221907]
Float32[59.22129, 21.958002]
Float32[44.346222, 33.920628]
Float32[41.461746, 46.05463]
Float32[22.226627, 48.81053]
Float32[22.226627, 12.77576]
Float32[20.96861, -3.9694731]
Float32[0.050881494, -11.29114]
Float32[-24.389984, -9.4113245]
Float32[-38.457962, -1.2922641]
Float32[-38.457962, 1.5510701]
Float32[-38.457962, 21.31275]
Float32[-46.853474, 22.066425]
Float32[-53.310215, 17.15966]
```

https://user-images.githubusercontent.com/32143268/167336172-a47f8bbb-0952-486f-a366-f51ece433c3d.mov

Note how the selected point seems to stop going near the edges of the plot.  A similar thing can be seen for other projections.

cc @Datseris - this PR should, once complete, hopefully solve the select_point issue.  I suspect that I might have to implement my own select_point function if the issue is in the projection.

### CODE REVIEW

**Comment by Datseris** [comment]
Great, thanks a lot! Tag me again and I'll try it out!

**Comment by asinghvi17** [comment]
https://github.com/JuliaPlots/Makie.jl/issues/1978 has some good info - looks like this is somewhat a Makie-end bug.  Will try that at some point and revert, but it might not be for a while.

**Comment by Datseris** [comment]
It's odd that we're tested explicitly at 1.6. Maybe we should change 1.6 to 1 in `ci.yml`, unless 1.6 is on purpose?

**Comment by ronisbr** [comment]
Hi! Is there any block holding this PR so that the labels in the longitude get fixed?

File: src/utils.jl [hunk 1]
```diff
@@ -62,7 +62,7 @@ end
 function Makie.inverse_transform(trans::Proj.Transformation)
     itrans = Base.inv(trans)
     return Makie.PointTrans{2}() do p
-        return Makie.apply_transform(itrans, p) .* PROJ_RESCALE_FACTOR
+        return Makie.apply_transform(itrans, p * PROJ_RESCALE_FACTOR) .* PROJ_RESCALE_FACTOR
```
Comment (asinghvi17) [inline src/utils.jl:65]:
```suggestion
        return Makie.apply_transform(itrans, p) .* PROJ_RESCALE_FACTOR

**Comment by asinghvi17** [comment]
As of now this PR just fixes the tick labels and switches to Proj from Proj4.  I'll try to do the ticklabels in a separate PR.

**Pull request merged**